docs/virtual_machines/network_binding_plugins.md

```diff
@@ -139,22 +139,43 @@ the binding.
 Registration includes the addition of the binding name with all its parameters
 to the Kubevirt CR.
 
-The following (optional) parameters are currently supported (as of v1.1.1):
-
-- networkAttachmentDefinition: Use the <namespace/object-name> format to specify
-  the [NetworkAttachementDefinition](https://github.com/k8snetworkplumbingwg/multi-net-spec/blob/master/v1.0/%5Bv1%5D%20Kubernetes%20Network%20Custom%20Resource%20Definition%20De-facto%20Standard.md)
-  that defines the CNI plugin and the configuration the binding plugin uses.
-  Used when the binding plugin needs to change the pod network namespace.
-- sidecarImage: Specify a container image in a registry.
-  Used when the binding plugin needs to modify the domain vNIC configuration
-  or when a service needs to be executed (e.g. DHCP server).
-- domainAttachmentType: Specify the name of a core domain attachment type.
-  A possible alternative to a sidecar, to configure the domain vNIC.
-  At the moment (v1.1.1) only a single type is supported: `tap`.
-
-  When both the `domainAttachmentType` and `sidecarImage` are specified,
-  the domain will first be configured according to the `domainAttachmentType`
-  and then the `sidecarImage` may modify it.
+The following (optional) parameters are currently supported:
+
+- [networkAttachmentDefinition](#networkattachmentdefinition)
+- [sidecarImage](#sidecarimage)
+- [domainAttachmentType](#domainattachmenttype)
+
+#### networkAttachmentDefinition
+From: v1.1.0
+
+Use the <namespace/object-name> format to specify
+the [NetworkAttachementDefinition](https://github.com/k8snetworkplumbingwg/multi-net-spec/blob/master/v1.0/%5Bv1%5D%20Kubernetes%20Network%20Custom%20Resource%20Definition%20De-facto%20Standard.md)
+that defines the CNI plugin and the configuration the binding plugin uses.
+Used when the binding plugin needs to change the pod network namespace.
+
+#### sidecarImage
+From: v1.1.0
+
+Specify a container image in a registry.
+Used when the binding plugin needs to modify the domain vNIC configuration
+or when a service needs to be executed (e.g. DHCP server). 
+
+#### domainAttachmentType
+From: v1.1.1
+
+The Domain Attachment type is a pre-defined core kubevirt method of
```

Comment:
Seems that the sentence is cropped/redundant.

Reply:
Done